lib/config/options/index.ts

```diff
@@ -589,6 +589,14 @@ const options: RenovateOptions[] = [
     stage: 'branch',
     type: 'string',
   },
+  {
+    name: 'npmrcMerge',
+    description:
+      'Whether to merge config npmrc with repo .npmrc content if both are found.',
```

Comment:
```suggestion
      'Whether to merge config.npmrc with repo .npmrc content if both are found.',
```

Not sure which is correct, so please check this suggestion. 😉